### PR TITLE
CAT-1532 Added Media Library in Objects Looks and Sounds.

### PR DESCRIPTION
**Pull request by loibi93**
The UX Team reviewed my work and said everything is ok !

### CODE REVIEW

**Comment by loibi93** [comment]
Rerun of failed tests:

https://jenkins.catrob.at/view/All-Categories/view/Catroid/job/Catroid-CustomEmulator-Single-ClassAndPackage-Test/499/

https://jenkins.catrob.at/view/All-Categories/view/Catroid/job/Catroid-CustomEmulator-Single-ClassAndPackage-Test/502/

https://jenkins.catrob.at/view/All-Categories/view/Catroid/job/Catroid-CustomEmulator-Single-ClassAndPackage-Test/503/

**Comment by robertpainsi** [comment]
retest this please

(Lets see if there are any new pmd warnings which have been added recently :wink:)

**Comment by loibi93** [comment]
ok to test

**Comment by avihric** [comment]
Remove the `solo.sleep(TIME_TO_WAIT);` in
* LookFragmentTest.testGetImageFromMediaLibrary
* SoundFragmentTest.testGetSoundFromMediaLibrary
* SpritesListFragmentTest.testGetSpriteFromMediaLibrary

as you use `solo.waitForText()`, which is sufficient.

File: catroidTest/src/org/catrobat/catroid/uitest/ui/fragment/SpritesListFragmentTest.java [hunk 1]
```diff
@@ -270,6 +278,57 @@ private void clickOnContextMenuItem(String spriteName, String menuItemName) {
...
+		while (Utils.isNetworkAvailable(getActivity())) {
+			solo.sleep(2000);
+			if (retryCounter > 30) {
+				break;
+			}
+			retryCounter++;
+		}
+		retryCounter = 0;
+		UiTestUtils.clickOnBottomBar(solo, R.id.button_add);
+		solo.sleep(TIME_TO_WAIT);
```
Comment (avihric) [inline catroidTest/src/org/catrobat/catroid/uitest/ui/fragment/SpritesListFragmentTest.java]: remove this line

File: catroidTest/src/org/catrobat/catroid/uitest/ui/fragment/SoundFragmentTest.java [hunk 2]
```diff
@@ -484,6 +490,101 @@ public void testAddSound() {
...
+		while (Utils.isNetworkAvailable(getActivity())) {
+			solo.sleep(2000);
+			if (retryCounter > 30) {
+				break;
+			}
+			retryCounter++;
+		}
+		retryCounter = 0;
+		UiTestUtils.clickOnBottomBar(solo, R.id.button_add);
+		solo.sleep(TIME_TO_WAIT);
```
Comment (avihric) [inline catroidTest/src/org/catrobat/catroid/uitest/ui/fragment/SoundFragmentTest.java]: remove this line

File: catroidTest/src/org/catrobat/catroid/uitest/ui/fragment/LookFragmentTest.java [hunk 3]
```diff
@@ -378,6 +388,98 @@ public void testShowAndHideDetails() {
...
+		while (Utils.isNetworkAvailable(getActivity())) {
+			solo.sleep(2000);
+			if (retryCounter > 30) {
+				break;
+			}
+			retryCounter++;
+		}
+		retryCounter = 0;
+		UiTestUtils.clickOnBottomBar(solo, R.id.button_add);
+		solo.sleep(TIME_TO_WAIT);
```
Comment (avihric) [inline catroidTest/src/org/catrobat/catroid/uitest/ui/fragment/LookFragmentTest.java]: remove this line

**Comment by loibi93** [comment]
No pmd warnings ! Should i retest the three failed tests ? They already passed in the last run !

**Comment by loibi93** [comment]
ok to test

**Comment by loibi93** [comment]
ok to test

**Comment by loibi93** [comment]
ok to test

**Comment by robertpainsi** [comment]
> Lets see if there are any new pmd warnings which have been added recently

Nonsense. What I wanted to say:
Lets see if there are any pmd warnings because we recently added new pmd checks.

**Pull request merged**